src/views/Layout.css

```
@@ -59,4 +59,52 @@
 .Nav-link.active {
 	text-decoration-thickness: 0.22em;
 	text-underline-offset: 0.1em;
+} */
+
+.tooltiptext {
+	color: white;
+	background-color: #192733;
+	font-size: 12px;
+	width: 120px;
+	visibility: hidden;
+	position: absolute;
+	z-index: 2;
+	border-radius: 10px;
+	padding: 6px 0;
+}
+
+.tooltiptext.icon {
+	top: 25px;
+}
+
+.tooltiptext.token {
+	top: -40px;
+}
+
+.tooltiptext::before {
+	background-color: #192733;
+	content: '';
+	position: absolute;
+	z-index: 1;
+	transform: rotate(45deg);
+	padding: 5px;
+}
+
+.tooltiptext.icon::before {
+	top: -5%;
+	left: 45%;
+}
+
+.tooltiptext.token::before {
+	top: 90%;
+	left: 45%;
+}
+
+.tooltip:hover .tooltiptext {
```

Typically, for accessibility, whenever you have :hover styles, you also want to apply them on :focus, so that folks who navigate the page with their keyboards can still see the change in styles.

I think this should work (but it's worth testing it out locally to double-check).

```suggestion
.tooltip:hover .tooltiptext,
.tooltip:focus .tooltiptext {
```